extension/awsproxy/factory_test.go

```diff
@@ -81,10 +82,14 @@ func TestFactory_CreateExtension(t *testing.T) {
 	err = ext.Start(ctx, mh)
 	assert.NoError(t, err)
 
-	resp, err := http.Post(
-		"http://"+address+"/GetSamplingRules",
-		"application/json",
-		strings.NewReader(`{"NextToken": null}`))
+	var resp *http.Response
+require.Eventually(t, func() bool {
```

Comment:
Looks like indentation needs fixing.

Reply:
Updated and also added failure message in the server fails to be ready by the time polling finishes.